processing/src/main/java/org/apache/druid/segment/QueryableIndexIndexableAdapter.java

```diff
@@ -280,7 +281,7 @@ public boolean hasTimeAndDimsChangedSinceMark()
     @Override
     public void close()
     {
-      CloseQuietly.close(closer);
+      CloseableUtils.closeAndWrapExceptions(closer);
```

Comment:
nit: you can modify the signature of this method to throw IOExceptoin, but this should also be fine.

Reply:
The superclass won't allow it. (It overrides `close` and declares it to throw nothing.) I guess I could change the superclass, but I didn't want to go that far.